bodhi/server/templates/update.html

```diff
@@ -40,7 +40,7 @@
 
 <script type="text/javascript">
   $(document).ready(function() {
-    var base_url = '${request.registry.settings["resultsdb_api_url"]}' + '/api/v1.0/';
+    var base_url = '${request.registry.settings["resultsdb_api_url"]}' + '/api/v2.0/';
```

Comment:
Very nice, I've been meaning to do this for a while. @ryanlerch, you are my hero!